website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-supported-dbt-versions.md

```diff
@@ -1,13 +1,29 @@
 ---
-title: "Supported dbt versions"
-id: "cloud-supported-dbt-versions"
+title: "Configuring dbt version"
+id: "cloud-configuring-dbt-version"
 ---
 
-dbt Cloud supports the three latest minor releases of dbt Core. At the present time, the following versions of dbt are supported:
+In addition to name and type, every environment in dbt Cloud has a 'dbt version' parameter. To set this, clicking the **edit** button on the environment's settings page, then the 'dbt version' drop-down bar.
+
+<img src="/static/img/dbt-cloud-environment-parameters.png" title="General settings of an environment"/>
```

Comment:
I think we need to drop `/static` from these img srcs. Let's also use the Lightbox component to make the images expandable on click :)

```suggestion
<Lightbox src="img/dbt-cloud-environment-parameters.png" title="General settings of an environment"/>
```

Same goes for the link below too!

Reply:
Interestingly, `/img/...` works but not `img/`, the latter of which requires `static/` before it--this is why I had static there in the first place. How odd...